### ISSUE TITLE
Support for Linux

### ISSUE THREAD

**Issue by oxr463** [LucidVR/opengloves-driver#202]
I'm running SteamVR on Ubuntu with my Valve Index.

**Comment by CosmicFusion** [comment]
Yeah I would love to see this on Linux too... , And I'll imagine it to be a lot easier to implement than windows .

**Comment by danwillm** [comment]
This is something I hope to work on this summer, including a port of opengloves to monado.

**Comment by jurassicjordan** [comment]
> This is something I hope to work on this summer, including a port of opengloves to monado.

I cant want to see it. If you can make it compatible with ALVR (not certain if there needs to be any compatibility requirements) that would be majorly awesome

**Comment by danwillm** [comment]
Opengloves is now supported in monado, here: https://gitlab.freedesktop.org/monado/monado/-/tree/main/src/xrt/drivers/opengloves

Will keep this issue open for now as I still plan on making the openvr driver linux compatible.
